api/src/main/java/org/pmiops/workbench/actionaudit/ActionAuditSpringConfiguration.kt

```diff
@@ -10,12 +10,16 @@ import java.util.UUID
 
 @Configuration
 open class ActionAuditSpringConfiguration {
+    companion object {
```

Comment:
this basically means "what's in here is static"

Reply:
why can't we convert this to java? It's only a spring config bean I think.

Reply:
I have not yet evaluated all files.  I agree that this one looks simple, so I'll probably get to it soon.